src/main.js

```diff
@@ -98,11 +98,17 @@ module.exports = {
         }
         const filePath = textEditor.getPath()
         const showRule = atom.config.get('linter-eslint.showRuleIdInMessage')
+        let rules = {}
+
+        if (textEditor.isModified()) {
+          rules = idsToIgnoredRules(atom.config.get('linter-eslint.silenceWhileTypingRules'))
```

Comment:
Maybe it would be more clear to call this variable `ignoredRules`?  I imagine this might be confusing for others approaching the code without knowing the background of why it's being done.